tests/acceptance/features/apiGraph/getUser.feature

```diff
@@ -1005,3 +1005,223 @@ Feature: get users
       | Guest       | User        |
       | Guest       | Guest       |
       | Guest       | Admin       |
+
+  @issue-6017
+  Scenario Outline: admin user gets the drive information of a user with different user role
+    Given the administrator has assigned the role "Admin" to user "Alice" using the Graph API
+    And the administrator has assigned the role "<userRole>" to user "Brian" using the Graph API
+    When user "Alice" gets the personal drive information of user "Brian" using Graph API
```

Comment:
before when action -> Brian should logs in to create personal space.
Put please whatever step for Brian